packages/core/src/local-admin-api.ts

```diff
@@ -32,7 +38,36 @@ export class LocalAdminApi implements AdminApi {
 
   async startIndexingModelData(modelData: Array<ModelData>): Promise<void> {
     await this.indexApi.indexModels(modelData)
-    await this.syncApi.startModelSync(modelData.map((idx) => idx.streamID.toString()))
+
+    if (this.syncApi.enabled) {
+      const models = await Promise.all(modelData.map(({ streamID }) => this.loadStream(streamID)))
+
+      const oldestModel = models.reduce((oldestModel, currentModel) => {
+        const oldestModelTimestamp = oldestModel.state.log[0].timestamp || Infinity
+        const currentModelTimestamp = currentModel.state.log[0].timestamp || Infinity
+
+        if (currentModelTimestamp < oldestModelTimestamp) {
+          return currentModel
+        }
+
+        return oldestModel
+      })
+
+      let syncOptions: ModelSyncOptions = {}
+      if (oldestModel.state.anchorProof) {
+        syncOptions = {
+          startBeforeTx: {
+            txHash: convertCidToEthHash(oldestModel.state.anchorProof.txHash),
+            numberOfBlocksBeforeTx: NUMBER_OF_BLOCKS_BEFORE_TX_TO_START_SYNC,
+          },
+        }
+      }
+
+      await this.syncApi.startModelSync(
```

Comment:
how do we handle Models that aren't anchored?  Also what if there's a mix of some Models that are anchored and some that are not?